src/plonemeeting/portal/core/migrations/migrate_to_1003.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from imio.migrator.migrator import Migrator
+from plonemeeting.portal.core.config import DEFAULT_CATEGORY_IA_DELIB_FIELD
+
+import logging
+
+logger = logging.getLogger("plonemeeting.portal.core")
+
+
+class MigrateTo1003(Migrator):
```

Comment:
Est ce qu'une migration est nécessaire?  À partir du moment où c'est la valeur par défaut qui est appliquée, je ne pense pas?  Sur une ancienne instance de test, çà fonctionne sans lancer l'upgrade step...  Si on la garde ce qui n'est pas bien grave, peut-être faire un "if institution.delib_category_field is None" pour éviter par exemple si on relance la step sans le faire exprès par après, de perdre ceux qui ont switché vers "classifier"

Reply:
J'ai eu un cas où la default value n'était pas retournée en local et ça plantait la synchro. Du coup pour pas prendre de risque ...

Reply:
Alors rajoute simplement le "if ..." pour ne pas tout casser par inadvertance dans 2 ans...

Reply:
J'ai retesté en recréant un site proprment depuis le master. la default fonctionne bien